GabrielaRamirez/Hackerrank/JavaIfElse.java

```diff
@@ -0,0 +1,39 @@
+import java.io.*;
```

Comment:
Please, avoid wildcard into imports
https://tharakamd-12.medium.com/is-it-bad-to-use-wildcard-imports-in-java-1b46a863b2be